ee/modules/110-istio/templates/control-plane/validatingwebhook-global.yaml

```diff
@@ -0,0 +1,46 @@
+# Helm cannot deploy the DestinationRule and PeerAuthentication manifests for tlsMode parameter because it stumbles on this webhook, so if isGlobalRevisionIstiodReady is false,
+# the webhook will be removed by the discovery_istiod_health.go hook.
```

Comment:
```suggestion
# Helm cannot deploy the DestinationRule and PeerAuthentication manifests for tlsMode parameter because it stumbles on this webhook.
# isGlobalRevisionIstiodReady is under discovery_istiod_health.go hook control.
```